tests/run_tests.sh

```diff
@@ -1 +1,2 @@
-
+INCLUDED_NBS=$(find ./nbdev_nbs -name "*.ipynb")
```

Comment:
should we rename `nbdev_nbs` to `nbs` then? 
also, there's still a file `nbdev.yaml` (referenced in `_quarto.yml`) .. could that be deleted?

Reply:
Will remove

Reply:
nbdev_nbs -> nbs_tests